ui/src/main/webapp/src/app/misc/about.component.ts

```diff
@@ -0,0 +1,30 @@
+import {Component, OnInit, AfterViewInit} from "@angular/core";
+
+@Component({
+    templateUrl: './about.component.html',
+    styles: [`
+        /deep/ img.windupContributor {
+            width: 35px;
+            margin: 4px;
+        }
+    `]
+})
+export class AboutPageComponent implements OnInit, AfterViewInit {
+
+    ngOnInit():any {
+    }
+
+    ngAfterViewInit(): void {
+        var divTarget = $("#windup-contributors");
```

Comment:
It may cause issues when it mixes with angular. Here it's an isolated single purpose functionality. And it is ready, and works. Let's ask Tobias whether we should invest in rewriting working things just to have them in current cool framework.